src/aiida/orm/nodes/data/remote/base.py

```diff
@@ -285,7 +285,13 @@ def _get_size_on_disk_du(self, full_path: Path, transport: Transport) -> int:
         """
 
         try:
-            retval, stdout, stderr = transport.exec_command_wait(f'du -s --bytes {full_path}')
+            # Initially, we were using the `--bytes` option here. However, this is equivalent to `--apparent-size
+            # --block-size=1`, with the `apparent-size` option being rather fragile (e.g. its implementation changed
+            # between Ubuntu 22.04 and 24.04, causing the tests to fail). Thus, we retain only `--block-size=1` to
```

Comment:
```suggestion
            # between different version of `du` of Ubuntu 22.04 and 24.04, causing the tests to fail). Thus, we retain only `--block-size=1` to
```

Might be more precise?

Reply:
Done!